docs/manual/common/guide/devmode/CassandraServer.md

```diff
@@ -44,11 +44,11 @@ In sbt:
 
 @[cassandra-clean-on-start](code/build-cassandra-opts.sbt)
 
-# Keyspace
+## Keyspace _(deprecated)_
```

Comment:
Part of the difficulty is that this page appears earlier in the documentation than the other page on Cassandra persistence. So I wanted to retain enough context that the deprecated content still makes sense, while moving most of it to the primary page on the topic.

IMHO, the documentation would be better organized by putting "Running Lagom in development" _after_ all of the sections on implementing services, right before "Running Lagom in production". Of course, that's a much bigger change that I would want to tackle separately.